### DISCUSSION TITLE
How to sync specific version helm chart using CLI

### DISCUSSION THREAD

**Discussion by naresh-naga** [argoproj/argo-cd#9744] [General]
I have argocd application that is set to manual sync and uses helm. while setting up, chart version is need so i gave it as 1.0.0. Now i have new version of chart 2.0.0 how to do i sync/deploy this version to argocd using CLI?
the sync cmd does not take version? i am trying to trigger ( push ) from jenkins server

**Comment by jannfis** [comment]
You need to update the `targetRevision` field in the application spec. You can do that using the CLI with the `argocd app set --revision` command. Your Argo CD user will need `update` RBAC permissions for this to work.

If Argo CD itself manages the `Application` resource (e.g. in an app-of-apps pattern), you will need to make this change in Git instead.

**Reply by naresh-naga** [reply]
thanks you!. If i use the cli to update the revision will it also update github file with this version?
